scripts/modules/b2b-account/quotes.js

```diff
@@ -20,7 +20,12 @@ define([
     function ($, api, _, Hypr, Backbone, HyprLiveContext,
     CustomerModels, CartModels, B2BAccountModels, ProductModalViews,
     ProductPicker, ProductModels, WishlistModels, MozuGrid, MozuGridCollection,
-        PagingViews, EditableView, QuoteModels) {
+    PagingViews, EditableView, QuoteModels) {
+        var nameFilter = "name cont ";
+        var expirationDateFilter  = "expirationdate ge ";
+        var timeComponent = "T00:00:00z";
+        var FILTERSTRING = "";
```

Comment:
This is not a constant so name it `filterString` or similar. The other variables are actually constants can could be named in capital case like `NAME_FILTER` and `TIME_COMPONENT`.

Reply:
done